### ISSUE TITLE
can not define self loss

### ISSUE THREAD

**Issue by haoxurt** [zhanghang1989/PyTorch-Encoding#349]
Thanks for share. When I try add some loss such as focal loss, I encounter the following error.Even if I realize CrossEntropy by self, the error is existed.  Hoping your help!
